datafusion/sqllogictest/test_files/spark/string/ascii.slt

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+query I
+SELECT spark_ascii('234');
+----
+50
+
+query I
+SELECT spark_ascii('');
+----
+0
+
+query I
+SELECT spark_ascii('222');
+----
+50
+
+query I
+SELECT spark_ascii(2::INT);
+----
+50
+
+query I
+SELECT spark_ascii(a) FROM (VALUES ('Spark'), ('PySpark'), ('Pandas API')) AS t(a);
+----
+83
+80
+80
```

Comment:
There appears to be a (likely unintended) change to the parquet-testing submodule in this PR too:

![Screenshot 2025-03-16 at 6 52 10 AM](https://github.com/user-attachments/assets/ba6cda29-fc58-482e-8819-4cad0f7ae967)